Cannot find module '@kurkle/color' or its corresponding type declarations.

### Expected behavior

I'm supporting to apply CHART.JS 4 compatibility to [treemap](https://github.com/kurkle/chartjs-chart-treemap/pull/165) controller.

I'm expecting that the import of chart.js/helpers is working properly when I'm testing the TS types of the controller.


### Current behavior

The test of TS types of the controller failed with the following error:

```
npm run test-types

> chartjs-chart-treemap@2.1.3 test-types
> tsc -p types/tests/

node_modules/chart.js/dist/helpers.d.ts:1:23 - error TS2307: Cannot find module '@kurkle/color' or its corresponding type declarations.

1 import { Color } from '@kurkle/color';
                        ~~~~~~~~~~~~~~~


Found 1 error in node_modules/chart.js/dist/helpers.d.ts:1
```

The [test code](https://github.com/kurkle/chartjs-chart-treemap/blob/main/types/tests/options.ts):

```ts
import '../index.esm';
import { Chart } from 'chart.js';
import { color as colorLib } from 'chart.js/helpers';

function colorFromValue(value: number, border?: boolean) {
  let alpha = (1 + Math.log(value)) / 5;
  const color = 'purple';
  if (border) {
    alpha += 0.01;
  }
  return colorLib(color)
    .alpha(alpha)
    .rgbString();
}
.... // not all
```


It seems we are obliged to add the dependency to kurkle/color,In version 3, the types were providing a "wrapper" con Color class and the direct dependency wasn't needed. Now the helpers are forcing to add the dependency.

As workaround, (thanks @LeeLenaleee) we added   `"skipLibCheck": true,` to tsconfig, but maybe it sounds to me just a workaround.

### Reproducible sample

npm run test-types (checking out PR https://github.com/kurkle/chartjs-chart-treemap/pull/165)

### Optional extra steps/info to reproduce

_No response_

### Possible solution

_No response_

### Context

_No response_

### chart.js version

v4.0.1

### Browser name and version

_No response_

### Link to your project

https://github.com/kurkle/chartjs-chart-treemap

--- a/package.json
+++ b/package.json
@@ -68,8 +68,10 @@
         "test-ci-karma": "cross-env NODE_ENV=test karma start ./karma.conf.cjs --auto-watch --single-run --coverage --grep",
         "test-ci-integration": "pnpm --filter \"./test/integration/**\" test"
     },
+    "dependencies": {
+        "@kurkle/color": "^0.3.0"
+    },
     "devDependencies": {
-        "@kurkle/color": "^0.2.1",
         "@rollup/plugin-commonjs": "^23.0.2",
         "@rollup/plugin-inject": "^5.0.2",
         "@rollup/plugin-json": "^5.0.1",

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -67,6 +67,7 @@ export default [
       'dist/helpers': 'src/helpers/index.ts'
     },
     plugins: plugins(),
+    external: _ => (/node_modules/).test(_),
     output: {
       dir: './',
       chunkFileNames: 'dist/chunks/[name].js',

